documentation/homework/todo-app/src/components/TasksList.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export default class TasksList extends React.Component {
+    constructor() {
+        super();
+    }
+
+    remove(e) {
+        const task = e.target.parentNode.querySelector('span').textContent;
```

Comment:
You don't need to access the DOM directly like this. Just use the **index** of the element.